deluge/ui/gtkui/path_combo_chooser.py

```diff
@@ -524,23 +534,23 @@ def popup(self):
 
         """
         # Entry is not yet visible
-        if not self.path_entry.flags() & gtk.REALIZED:
+        if not (self.path_entry.get_realized()):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary parens after 'not' keyword](https://www.codacy.com/app/tbkizle/deluge/file/2628318269/issues/source?bid=3238201&fileBranchId=3238205#l537)